### PR TITLE
PLATML -  4147

### PR DESCRIPTION
**Pull request by tumulurik**

### CODE REVIEW

File: recipes/scala/src/main/scala/com/adobe/platform/ml/Evaluator.scala [hunk 1]
```diff
@@ -0,0 +1,62 @@
+/*#####################################################################
+# ADOBE CONFIDENTIAL
+# ___________________
+#
+#  Copyright 2018 Adobe
```
Comment (rjrussell77) [inline recipes/scala/src/main/scala/com/adobe/platform/ml/Evaluator.scala]: Brand new file?  2019?

File: recipes/scala/src/main/scala/com/adobe/platform/ml/Evaluator.scala [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+import java.util
+
+import com.adobe.platform.ml.sdk.MLEvaluator
+import com.adobe.platform.ml.sdk.MLMetric
+import com.adobe.platform.ml.config.ConfigProperties
+import org.apache.spark.ml.Transformer
+import org.apache.spark.sql.DataFrame
+import org.apache.spark.sql.functions._
+
+
```
Comment (sathyarun18) [inline recipes/scala/src/main/scala/com/adobe/platform/ml/Evaluator.scala:32]: add javadocs

**Review by sathyarun18 — APPROVED** [review]

File: recipes/scala/src/main/scala/com/adobe/platform/ml/Evaluator.scala [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+class Evaluator extends MLEvaluator {
+
+  override def split(configProperties:ConfigProperties, data: DataFrame): (DataFrame, DataFrame) = {
+    val dataSplit = data.randomSplit(Array(0.8, 0.2), seed = 11L)
+    (dataSplit(0), dataSplit(1))
+
+  }
+
+  def evaluate(configProperties: ConfigProperties, model: Transformer, dataFrame: DataFrame): util.ArrayList[MLMetric] = {
```
Comment (rjrussell77) [inline recipes/scala/src/main/scala/com/adobe/platform/ml/Evaluator.scala]: Do we have any standard formatter?  This line extends pretty far out and may be brought in given shorter chars-per-line policy.

File: recipes/scala/src/main/scala/com/adobe/platform/ml/Evaluator.scala [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+import com.adobe.platform.ml.config.ConfigProperties
+import org.apache.spark.ml.Transformer
+import org.apache.spark.sql.DataFrame
+import org.apache.spark.sql.functions._
+
+
+class Evaluator extends MLEvaluator {
+
+  override def split(configProperties:ConfigProperties, data: DataFrame): (DataFrame, DataFrame) = {
+    val dataSplit = data.randomSplit(Array(0.8, 0.2), seed = 11L)
```
Comment (rjrussell77) [inline recipes/scala/src/main/scala/com/adobe/platform/ml/Evaluator.scala:43]: Can or should the 80/20 split be passed in via the `configProperties`?

**Review by jonaphin — APPROVED** [review]
looks good, thank you for adding the evaluator!

**Review by rjrussell77 — APPROVED** [review]
LGTM - couple comments

File: recipes/scala/src/main/scala/com/adobe/platform/ml/Evaluator.scala (hunk 2 above)
Comment (tumulurik) [reply recipes/scala/src/main/scala/com/adobe/platform/ml/Evaluator.scala:32]: Added javadocs

File: recipes/scala/src/main/scala/com/adobe/platform/ml/Evaluator.scala (hunk 1 above)
Comment (tumulurik) [reply recipes/scala/src/main/scala/com/adobe/platform/ml/Evaluator.scala]: Yes, updated

File: recipes/scala/src/main/scala/com/adobe/platform/ml/Evaluator.scala (hunk 3 above)
Comment (tumulurik) [reply recipes/scala/src/main/scala/com/adobe/platform/ml/Evaluator.scala]: Currently, have the setting at 120

File: recipes/scala/src/main/scala/com/adobe/platform/ml/Evaluator.scala (hunk 4 above)
Comment (tumulurik) [reply recipes/scala/src/main/scala/com/adobe/platform/ml/Evaluator.scala:43]: This is a normal ratio in which the data is split.

**Review by tatsuyawi — APPROVED** [review]

**Pull request merged**